pxtcompiler/emitter/snippets.ts

```diff
@@ -311,11 +311,13 @@ namespace ts.pxtc.service {
                 if (typeNode.kind === SK.StringKeyword || deflKind === SK.StringKeyword) {
                     return paramDefl.indexOf(`"`) != 0 ? `"${paramDefl}"` : paramDefl;
                 }
-
+                const type = checker?.getTypeAtLocation(param);
+                const typeSymbol = getPxtSymbolFromTsSymbol(type?.symbol, apis, checker);
+                if (typeSymbol?.attributes.fixedInstances) {
+                    return paramDefl;
```

Comment:
Do you need to check if we're in python here? Does paramDefl work for both languages?

Reply:
oh for some reason I thought fixedInstances meant they had the same name in py/ts. I'll just remove this part

Reply:
fixedInstances are when we export a bunch of instances and use them to populate a dropdown. For example:

https://github.com/microsoft/pxt-common-packages/blob/master/libs/game/controlleroverrides.ts

Reply:
It's sorta like an enum, but it compiles differently. Those are for the controller blocks in arcade

Reply:
okay hmm looks like it's just a snakify? I will add